packages/k6-tests/src/koko/020-navigate-file-tree.md

```diff
@@ -0,0 +1,70 @@
+## Description
+The `navigate file tree` test is intended to see how the instance behaves when many users move through a configurable deep folder hierarchy at the same time.
+Since k6 does not use a browser for the tests, we propfind each folder to simulate the client behavior. 
```

Comment:
```suggestion
The `navigate file tree` test is intended to see how the instance behaves when many users move through a configurable deep folder hierarchy at the same time. Since k6 does not use a browser for the tests, a propfind for each folder to simulate the client behavior is used. 
```